packages/@pollyjs/adapter-puppeteer/src/index.js

```diff
@@ -1,10 +1,11 @@
 import Adapter from '@pollyjs/adapter';
+import { URL } from '@pollyjs/utils';
```

Comment:
Using `url-parse` instead of node's `URL` as it is a consistent API across node versions and the browser.